### PR TITLE
Miscellaneous: Minor commits from the job branch

### PR DESCRIPTION
**Pull request by alongd**
This PR is another in a chain meant to assist managing a large future PR that re-writes the job module.

Here, some minor commits were selected, and one simple addition to a test.

### CODE REVIEW

File: arc/species/conformers.py [hunk 1]
```diff
@@ -148,8 +148,7 @@ def generate_conformers(mol_list: Union[List[Molecule], Molecule],
         de_threshold (float, optional): Energy threshold (in kJ/mol) above which wells will not be considered.
         smeared_scan_res (float, optional): The resolution (in degrees) for scanning smeared wells.
         combination_threshold (int, optional): A threshold below which all combinations will be generated.
-        force_field (str, optional): The type of force field to use (MMFF94, MMFF94s, UFF, GAFF, fit).
-                                     'fit' will first run MMFF94, than fit a custom Amber FF to the species.
+        force_field (str, optional): The type of force field to use (MMFF94, MMFF94s, UFF, GAFF).
```
Comment (kfir4444) [inline arc/species/conformers.py:151]: So, there was never a fit option?

File: arc/species/converterTest.py [hunk 2]
```diff
@@ -4460,7 +4460,6 @@ def test_modify_coords(self):
                                           )
         self.assertAlmostEqual(calculate_dihedral_angle(coords=new_xyz, torsion=indices, index=1), 200, places=3)
 
-
```
Comment (kfir4444) [inline arc/species/converterTest.py:4463]: Missing one empty line between the functinos

**Review by kfir4444 — COMMENTED** [review]
@alongd I think the converterTest.py file is missing some empty lines between the functions. I marked it with comments.  Other than that, it's mainly cosmetic and seems good.

File: arc/species/converterTest.py [hunk 3]
```diff
@@ -4514,7 +4513,6 @@ def test_compare_confs(self):
         self.assertFalse(converter.compare_confs(ch4_1, ch4_3))
         self.assertAlmostEqual(converter.compare_confs(ch4_1, ch4_3, rmsd_score=True), 0.00044803, 5)
 
-
```
Comment (kfir4444) [inline arc/species/converterTest.py:4517]: also here

File: arc/species/converterTest.py [hunk 4]
```diff
@@ -4569,7 +4567,6 @@ def test_compare_confs(self):
         self.assertTrue(converter.compare_confs(occco_1, occco_3))
         self.assertAlmostEqual(converter.compare_confs(occco_1, occco_3, rmsd_score=True), 0.0, places=4)
 
-
```
Comment (kfir4444) [inline arc/species/converterTest.py:4572]: also here

**Review by kfir4444 — APPROVED** [review]
Never mind, I thought it was between different functions. Approved 👍

File: arc/species/conformers.py (hunk 1 above)
Comment (alongd) [reply arc/species/conformers.py:151]: It never functioned correctly... It'll be deprecated in the next job PR, I removed the description here

**Pull request merged**